src/main/cli/cli.c

```diff
@@ -1242,30 +1242,36 @@ static void cliSerialPassthrough(char *cmdline)
         return;
     }
 
-    int id = -1;
-    uint32_t baud = 0;
+    int rightSidePortID = -1;
```

Comment:
Please use camelCase.

Also, `rightSide` / `leftSide` isn't very indicative of what this is doing. How about `input` or `inputPort` and `output` or `outputPort`?

Reply:
Yeah, got it.
About `rightSide`/`leftSide`, I thought serial passthrough is something like  pipe, it's a two-way communication channel, 'input' and 'output' look like are a one-way. even in VCP and uartX serial passthrough, the communication are two-way.

Reply:
yeah.  now the left side port cfg at the "right side", but right side port cfg at the "left side", this is confusing.
Yes, use `port1` and`port2` would be good, I'm sorry for I just complaint it, but did not give a good solution. Thank you!